lib/test-worker.js

```diff
@@ -234,7 +234,7 @@ adapter.installPrecompilerHook();
 
 try {
 	(opts.require || []).forEach(x => {
-		if (/[/\\]@std[/\\]esm[/\\]index\.js$/.test(x)) {
+		if (/[/\\](@std[/\\])?esm[/\\]index\.js$/.test(x)) {
```

Comment:
Given that `x` here is an absolute path I'm a little nervous about just looking for `/esm/`. That seems like it could yield false positives more often than `@std/esm` did.

It's probably fine if we'd check for `node_modules` as well. Alternatively perhaps `esm` could tag its exported function with a registered symbol that we can check for?

```js
const required = require(x)
try {
  if (required[Symbol.for('esm')]) {
    require = required(module)
  }
} catch (_) {}
```

Do you think we should drop support for `@std` here?

Reply:
Ah, yes I can make that more detectable!

Reply:
We have a symbol namespace we use currently (though not exposed on the exports).
It goes something like `"esm" + ZWJ + ":thing"` so for the exports I could expose a symbol named for `"esm"+ZWJ`, so without the `:thing` bit.

Reply:
I'll noodle around on it and report back.